packages/rum-core/test/utils/polyfill.js

```diff
@@ -93,3 +93,21 @@ Object.setPrototypeOf =
     obj.__proto__ = proto
     return obj
   }
+
+/**
+ * Polyfilling the CustomEvent since they are available as objects
+ * in IE 9-11
+ * https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent#Polyfill
+ */
+;(function() {
+  if (typeof window.CustomEvent === 'function') return false
+
+  function CustomEvent(event, params) {
+    params = params || { bubbles: false, cancelable: false, detail: null }
+    var evt = document.createEvent('CustomEvent')
+    evt.initCustomEvent(event, params.bubbles, params.cancelable, params.detail)
+    return evt
+  }
+
+  window.CustomEvent = CustomEvent
```

Comment:
Can we move this to the file level instead of setting on window? The reason being what we already discussed in other PRs in regards to avoiding global polyfills to make sure our tests run in a close environment to the real browsers!

Reply:
We wont run in to issues like those since this one is pretty specific to tests and we don't use this code on our src files.

Reply:
We don't currently `CustomEvent` in our source but we might in the future. I think it's a good idea to keep the polyfills to a minimum and limited to the file level as much as possible!

Reply:
moved it.

Reply:
moved it inside file.